src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -139,16 +141,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     String messageContent = request.getParameter("message");
-
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
```

Comment:
I am not sure I understand what need to be fixed here.

Reply:
all good